ssc/cmod_battwatts.cpp

```diff
@@ -241,6 +241,8 @@ battwatts_create(size_t n_recs, size_t n_years, int chem, int meter_pos, double
     }
     batt_vars->batt_dispatch_auto_can_charge = true;
     batt_vars->batt_dispatch_auto_can_gridcharge = true;
+    batt_vars->batt_dispatch_charge_only_system_exceeds_load = true;
+    batt_vars->batt_dispatch_discharge_only_load_exceeds_system = true;
```

Comment:
Perhaps this should be false so the ReOpt runs work better even in Battwatts

Reply:
Changed to false in latest push. Corresponding annual energy and lcoe changes pushed to paired SAM branch.